bahmni-proxy/resources/bahmni-proxy.conf

```diff
@@ -56,6 +56,10 @@ ProxyPassReverse /oviyam2 http://dcm4chee:8055/oviyam2
 ProxyPass /atomfeed-console http://atomfeed-console:9080/atomfeed-console
 ProxyPassReverse /atomfeed-console http://atomfeed-console:9080/atomfeed-console
 
+#Metabase 
+ProxyPass /metabase http://metabase:3000
```

Comment:
I see mb_site_url environment variable has not been added, so I hope this been tested properly

Reply:
Yes Priyanka